### ISSUE TITLE
Delete Incomprehensible speech (HP:0002546)

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#9725]
**HPO term**
Incomprehensible speech
HP:0002546

**Reason for deprecation**
This term is not specific and is better replaced by a term like "Abnormal communication" which connects to more specific child terms.

**Suggested replacement or term to consider**
This term could be merged into Abnormal communication or Abnormality of speech or vocalization. Since it is currently a child of the latter (HP:0002167), I would merge it there since it will be the least disruptive.

**Comment by pnrobinson** [comment]
@bcoleman96 This term is being used for a neurological abnormality -- see these annotated diseases: https://hpo.jax.org/app/browse/term/HP:0002546
I agree it does not add much, but will try to check whether it can be replaced by more specific terms such as dysarthria for some of the diseases or whether there is some other feature.

**Comment by bcoleman96** [comment]
Can we add a label for language to this, so we remember to discuss it at the next meeting for language?

**Comment by pnrobinson** [comment]
@bcoleman96 I think this is a justifiable term, it is a descriptive observation when we cannot break things down further (i.e., not necessarily the same as dysarthria), and this is a term that is used in neurology.
